docs/source/science/GEDI/GEDI_L4B.ipynb

```diff
@@ -2,33 +2,80 @@
  "cells": [
```

Comment:
<p>Is this a section title? Can we shorted and capitalize the important parts? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>It shouldn't be, I thought I changed it to normal text. Will go back and adjust!</p>